cypress/e2e/relatedAssets.feature

```diff
@@ -1,8 +1,8 @@
+@property
 @authentication
 @common
-@processes
-@property
 @root
+@search
```

Comment:
I don't know what this one is about, but I'm guessing it's the asset relation tree on the "view asset" MFE page.

If so, why do we need to run this spec when search MFE receives changes? Can search MFE break how that relation tree is displayed?